packages/bpk-docs/src/pages/BadgePage/BadgePage.js

```diff
@@ -43,7 +45,7 @@ const components = [
       <BpkBadge className={badgeClassName}>Apples</BpkBadge>,
       <BpkBadge className={badgeClassName}>Bananas</BpkBadge>,
       <BpkBadge className={badgeClassName}>Strawberries</BpkBadge>,
-      <BpkBadge>Pears</BpkBadge>,
+      <BpkBadge className={badgeClassName}>Pears</BpkBadge>,
```

Comment:
Shouldn't have removed the classname. Also, the classname should probably be on the other added ones too!